drivers/hmis/app/models/hmis/hud/processors/income_benefit_processor.rb

```diff
@@ -60,7 +60,13 @@ def income_source_attributes(amount_field, value)
       case @hud_values['IncomeBenefit.incomeFromAnySource']
       when 'YES'
         # If overall income was 1 (yes), then this specific income field must be 1 or 0 (yes or no)
-        bool_attribute_value = amount_attribute_value&.positive? ? 1 : 0
+        case amount_attribute_value
+        when Integer, Float, nil
+          bool_attribute_value = amount_attribute_value&.positive? ? 1 : 0
+        else
+          # the client may send non-numeric values
```

Comment:
I guess this is expected behavior. If not we should probably add
```suggestion
          Sentry.capture_message("Unexpected value \"#{amount_attribute_value}\" received for #{amount_attribute_name}")
```

Reply:
Agreed, we should add that. Since the frontend input component has been updated, It is not expected for these to be non-numeric.